NEWS.rst

```diff
@@ -8,6 +8,113 @@ https://twisted.org/trac/ticket/<number>
 
 .. towncrier release notes start
 
+Twisted 24.10.0rc1 (2024-10-19)
+===============================
+
+Features
+--------
+
+- Python 3.13 is now supported. (#11750)
+- twisted.internet.defer.succeed() is significantly faster, and awaiting Deferred has also been sped up. (#12225)
+- twisted.python.failure.Failure creation no longer records the place where it was created. This reduces creation time by 60% at least, thereby speeding up Deferred error handling. (#12230)
+- twisted.internet.defer.Deferred no longer removes the traceback object from Failures. This may result in more objects staying in memory if you don't clean up failed Deferreds, but it speeds up error handling and enables improvements to traceback reporting. (#12234)
+- twisted.internet.defer APIs are 2%-4% faster in many cases. (#12237)
+- twisted.internet.defer.Deferred runs callbacks with chained Deferreds a little faster. (#12284)
+- The reactor now will use a little less CPU when events have been scheduled with callLater(). (#12286)
+- Creation of twisted.python.failure.Failure is now faster. (#12288)
+- twisted.web.server is 1-2% faster in some cases. (#12318)
```

Comment:
Shouldn't this be down in the web section?

Reply:
True.

I will update this before the release.

---------

I would prefer to have just a single newsfragment directory for the whole project.

In recent years, twisted is mostly common async + HTTP + SSH

Many releases are done without changes to `words`, `mail` or `names` ... and `conch` and `trial` usually get one or two changes per release.